src/ploneintranet/suite/setuphandlers.py

```diff
@@ -174,7 +174,7 @@ def create_news_items(newscontent):
         # api.content.transition(obj=obj, transition='publish')
 
         obj.setEffectiveDate(newsitem['publication_date'])
-        obj.reindexObject(idxs=['effective', ])
+        obj.reindexObject(idxs=['effective', 'Subject', ])
```

Comment:
I had actually tried that too before I opened the ticket, but it did not work for me then. But now I tried it again, and all is fine. I must have overlooked something then.